src/celeritas/field/MakeMagFieldPropagator.hh

```diff
@@ -34,6 +34,7 @@ namespace celeritas
  */
 template<template<class EquationT> class StepperT, class FieldT>
 CELER_FUNCTION decltype(auto)
+// NOLINTNEXTLINE(cppcoreguidelines-missing-std-forward)
```

Comment:
OK this is really absurd... since we're forwarding with our own forward...

Reply:
yeah, it's specifically looking for a `std::forward` 😅